app/models/user.rb

```diff
@@ -32,6 +32,10 @@ class User < ActiveRecord::Base
 
   DEFAULT_TIMEZONE = "Eastern Time (US & Canada)"
 
+  def self.anonymous
+    new(first_name: 'Unknown', last_name: 'User', email_address: 'unknown@example.com')
```

Comment:
thoughts on using the null object pattern? https://robots.thoughtbot.com/rails-refactoring-example-introduce-null-object